Automattic/newspack-plugin
# chore: remove payments wizard

### All Submissions:






### Changes proposed in this Pull Request:



Removes the Payment Wizard, since Newspack now has a different approach to customer billing.

### How to test the changes in this Pull Request:

1. On a test site with both `NEWSPACK_STRIPE_PLAN` and `NEWSPACK_STRIPE_MODE` set in `wp-config.php` verify the Payment wizard is visible on `master`.
2. Switch to this branch, verify it is no longer visible and there are no errors.

### Other information:
